docs/how-to/use-rewards-api.md

```diff
@@ -0,0 +1,136 @@
+---
+description: Use the rewards API to view validator rewards.
+sidebar_position: 13
+---
+
+# View block rewards
+
+Use the [rewards API](https://consensys.github.io/teku/#tag/Rewards) to view the validator rewards for specific blocks.
```

Comment:
This does not apply to the Attestation rewards. It isn't for a block but for an epoch.

Reply:
we could just delete `for specific blocks`